x-pack/plugins/siem/public/components/open_timeline/timelines_table/common_columns.tsx

```diff
@@ -0,0 +1,120 @@
+import { EuiButtonIcon, EuiLink } from '@elastic/eui';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { FormattedDate } from '../../formatted_date';
+import { getEmptyTagValue } from '../../empty_value';
+import { isUntitled } from '../helpers';
+import { NotePreviews } from '../note_previews';
+import { OnOpenTimeline, OnToggleShowNotes, TimelineResult } from '../types';
+
+import * as i18n from '../translations';
+
+/** the width of the description column when showing extended columns */
+export const EXTENDED_COLUMNS_DESCRIPTION_WIDTH = '30%';
+
+export const DESCRIPTION_WIDTH = '45%';
+
+const ExpandButtonContainer = styled.div`
+  position: relative;
+  top -4px;
+`;
+
+/**
+ * Returns the column definitions (passed as the `columns` prop to
+ * `EuiBasicTable`) that are common to the compact `Open Timeline` modal view,
+ * and the full view shown in the `All Timelines` view of the `Timelines` page
+ */
+export const getCommonColumns = ({
+  itemIdToExpandedNotesRowMap,
+  onOpenTimeline,
+  onToggleShowNotes,
+  showExtendedColumnsAndActions,
+}: {
+  onOpenTimeline: OnOpenTimeline;
+  onToggleShowNotes: OnToggleShowNotes;
+  showExtendedColumnsAndActions: boolean;
+  itemIdToExpandedNotesRowMap: Record<string, JSX.Element>;
+}) => [
+  {
+    width: '32px',
+    isExpander: true,
+    render: ({ notes, savedObjectId }: TimelineResult) =>
+      notes != null && notes.length > 0 && savedObjectId != null ? (
+        <ExpandButtonContainer>
+          <EuiButtonIcon
+            data-test-subj="expand-notes"
+            onClick={() =>
+              itemIdToExpandedNotesRowMap[savedObjectId] != null
+                ? onToggleShowNotes(omit(savedObjectId, itemIdToExpandedNotesRowMap))
+                : onToggleShowNotes({
+                    ...itemIdToExpandedNotesRowMap,
+                    [savedObjectId]: <NotePreviews notes={notes} />,
+                  })
+            }
+            aria-label={itemIdToExpandedNotesRowMap[savedObjectId] ? i18n.COLLAPSE : i18n.EXPAND}
+            iconType={itemIdToExpandedNotesRowMap[savedObjectId] ? 'arrowDown' : 'arrowRight'}
+          />
+        </ExpandButtonContainer>
+      ) : null,
+  },
+  {
+    dataType: 'string',
+    field: 'title',
+    name: i18n.TIMELINE_NAME,
+    render: (title: string, timelineResult: TimelineResult) =>
+      timelineResult.savedObjectId != null ? (
+        <EuiLink
+          data-test-subj={`title-${timelineResult.savedObjectId}`}
+          onClick={() =>
+            onOpenTimeline({
+              duplicate: false,
+              timelineId: `${timelineResult.savedObjectId}`,
+            })
+          }
+        >
+          {isUntitled(timelineResult) ? i18n.UNTITLED_TIMELINE : title}
+        </EuiLink>
+      ) : (
+        <div data-test-subj={`title-no-saved-object-id-${timelineResult.title || 'no-title'}`}>
+          {isUntitled(timelineResult) ? i18n.UNTITLED_TIMELINE : title}
+        </div>
```

Comment:
Looks a bit weird that you use two different variables for title. The original off of `timelineResult.title` and then the other which is `title`

Reading the unit tests I assume there's no difference between the two but would be suspicious if I was maintaing this code.